notebook/spec2vec/tutorial.ipynb

```diff
@@ -284,8 +180,7 @@
     "    dir.create(\"matches\")\n",
     "}\n",
     "for (i in seq_along(spectra_matches)){\n",
-    "    matches <- data.frame(lapply(spectra_matches[i], as.character), stringsAsFactors=FALSE)\n",
-    "    write.csv(matches, sprintf(\"matches/spectrum_%s.csv\", i))\n",
+    "    write.csv(spectra_matches[[i]],sprintf(\"matches/spectrum_%s.csv\",i))\n",
```

Comment:
are you sure this works?? last time I tried I had to make those weird adjustments that are on the diff

Reply:
Nah, Idk why this changed.